### ISSUE TITLE
Accessibility run though

### ISSUE THREAD

**Issue by nickcolley** [pouchdb/pouchdb#3886]
Inspired by @charlotteis

 I imagine there's tons of low hanging fruit and some more interesting ones to run through accessibility wise, opening this up in-case someone else wants to have a go before I get to it.

**Comment by Charlotteis** [comment]
Let me know if you'd like a hand! :)

**Comment by nickcolley** [comment]
@Charlotteis going to have a first run, will reach out though if you want to review? :dancer:

**Comment by Charlotteis** [comment]
I would love to! Good luck :tada:

**Comment by nickcolley** [comment]
Just ran a little tool on it and looks like we have a few contrast issues:

![screenshot from 2015-05-29 22 30 10](https://cloud.githubusercontent.com/assets/2445413/7893117/7ced9a3e-0652-11e5-9fc7-9f9f9ef921bb.png)

Not sure what to do here because increasing the contrast makes everything look really awful even though it's more accessible. I know some sites offer a high-contrast mode but not sure if that's a good solution.

**Comment by nickcolley** [comment]
Not sure the tables on http://0.0.0.0:4000/adapters.html are going to much use in a screen reader judging by the headers and 'ticks'

**Comment by nickcolley** [comment]
Start on this: https://github.com/pouchdb/pouchdb/compare/master...nickcolley:3886-accessibility-run-through?w=1

Started the following:
- [] Added landmarks
- [] Ensure all links have text
- [] Made sure alt tags are not too wordy
- [] Made sure link text makes sense (no 'click here')

**Comment by nickcolley** [comment]
- [ ] Check what happens if a screenreader tries to read the code block, and potentially fix.

**Comment by Charlotteis** [comment]
@nickcolley be aware that `<main>` is not supported by any version of IE. If that is important to pouchdb then you need to keep the main element as a div with a role of main, or there are some other things you can do: http://weblog.west-wind.com/posts/2015/Jan/12/main-HTML5-Tag-not-working-in-Internet-Explorer-91011

**Comment by nickcolley** [comment]
@Charlotteis good to know, I'll swap it out I think.

**Comment by Charlotteis** [comment]
:+1:

**Comment by daleharvey** [comment]
@nickcolley hows this going? I would suggest if you get a whole bunch done to get it in and merged even if its not completely finished yet, easier than it going stale, this looks great though cheers :)

**Comment by nickcolley** [comment]
Hit some of the obvious stuff, I'm on holiday at the moment but will get something mergable when I get back.

**Comment by nickcolley** [comment]
Okay I've made a PR with some of the obvious stuff.

**Comment by daleharvey** [comment]
And PR landed, we can likely improve more but can be done as individual issues as needed, thanks
